src/WefactAPI.php

```diff
@@ -117,6 +117,23 @@ protected function sendRequest($controller, $action, $params)
             ];
         } else {
             $result = json_decode($curlResp, true);
+
+            if($result['action'] != 'list' || $result['currentresults'] > 0) {
```

Comment:
What does this `$result['action'] != 'list' ||` section add? I'm a little confused with this line vs `if($result['action'] == 'list') {` on 125